src/uu/cp/src/cp.rs

```diff
@@ -563,6 +563,7 @@ impl Options {
         let not_implemented_opts = vec![
             OPT_COPY_CONTENTS,
             OPT_SPARSE,
+            #[cfg(not(any(windows, unix)))]
```

Comment:
Unix implies Linux and Mac?

Reply:
I'm not sure ATM. I've only reflected what [WalkDir supports](https://docs.rs/walkdir/2.3.1/src/walkdir/util.rs.html#19)

Reply:
yeah, this is all OS pretty much :)
(except Fuschia)